tests/test_convert_threaded.py

```diff
@@ -125,72 +87,28 @@ def test_convert_s3_path_sqlite(
-                ".cytotable.mocks3.direct.parquet"
-            ),
-            dest_datatype="parquet",
-            chunk_size=100,
-            preset="cellprofiler_sqlite_pycytominer",
-            endpoint_url=example_s3_endpoint,
-            # use explicit cache to avoid temp cache removal / overlaps with
-            # sequential s3 SQLite files. See below for more information
-            # https://cloudpathlib.drivendata.org/stable/caching/#automatically
-            local_cache_dir=f"{fx_tempdir}/sqlite_s3_cache/1",
-        )
+    s3_result = convert(
+        source_path=example_s3_path_sqlite_jump,
+        dest_path=f"{fx_tempdir}/s3_test",
+        dest_datatype="parquet",
+        source_datatype="sqlite",
+        # set chunk size to amount which operates within
+        # github actions runner images and related resource constraints.
+        chunk_size=30000,
+        preset="cellprofiler_sqlite_cpg0016_jump",
+        sort_output=False,
+        no_sign_request=True,
+        # use explicit cache to avoid temp cache removal / overlaps with
+        # sequential s3 SQLite files. See below for more information
+        # https://cloudpathlib.drivendata.org/stable/caching/#automatically
+        local_cache_dir=f"{fx_tempdir}/sqlite_s3_cache/2",
     )
 
-    # s3 sqlite read with nested sqlite file
-    s3_cytotable_table_nested = parquet.read_table(
-        source=convert(
-            source_path="s3://example/nf1/",
-            dest_path=(
-                f"{fx_tempdir}/{pathlib.Path(data_dir_cellprofiler_sqlite_nf1).name}"
-                ".cytotable.mocks3.nested.parquet"
-            ),
-            dest_datatype="parquet",
-            chunk_size=100,
-            preset="cellprofiler_sqlite_pycytominer",
-            endpoint_url=example_s3_endpoint,
-            # use explicit cache to avoid temp cache removal / overlaps with
-            # sequential s3 SQLite files. See below for more information
-            # https://cloudpathlib.drivendata.org/stable/caching/#automatically
-            local_cache_dir=f"{fx_tempdir}/sqlite_s3_cache/2",
-        )
-    )
+    # read only the metadata from parquet file
+    parquet_file_meta = parquet.ParquetFile(s3_result).metadata
 
-    assert local_cytotable_table.sort_by(
-        [(name, "ascending") for name in local_cytotable_table.schema.names]
-    ).equals(
-        s3_cytotable_table.sort_by(
-            [(name, "ascending") for name in s3_cytotable_table.schema.names]
-        )
-    )
-    assert local_cytotable_table.sort_by(
-        [(name, "ascending") for name in local_cytotable_table.schema.names]
-    ).equals(
-        s3_cytotable_table_nested.sort_by(
-            [(name, "ascending") for name in s3_cytotable_table_nested.schema.names]
-        )
-    )
+    # check the shape of the data
+    assert (parquet_file_meta.num_rows, parquet_file_meta.num_columns) == (74226, 5928)
```

Comment:
It doesn't look like you're checking for any sorting issue here? Maybe this is good b/c it is focused, but I wonder if testing here would be good? I really don't know if this is a good suggestion!

Reply:
Thanks, this is a great question! After thinking about this and seeing issues in other testing without sorts I moved to retain sorted output (the default) by removing `sort_output=False` and reduced the number of columns needed to perform a join. I reduced the columns to help avoid memory issues I was seeing in GitHub Actions runners. This said, I feel that addressing the join sort memory consumption should be a new issue / PR to help reduce the overall resource requirements for CytoTable.